src/app/modules/backbone/backbone.common.ts

```diff
@@ -0,0 +1,27 @@
+import { Routes } from "@angular/router";
+
+import { AppComponent } from "@app/app.component";
+import { BackbonePageComponent } from "@modules/backbone/pages/backbone/backbone.page";
+import { BackboneComponent } from "@modules/backbone/components/backbone/backbone.component";
+
+export const pageDeclarations: any[] = [
+    BackbonePageComponent,
+];
+
+export const componentDeclarations: any[] = [
+    BackboneComponent
```

Comment:
No need for a backbone component here, we will add other components such as header/sidebar, footer